builtin/providers/google/resource_compute_instance_group.go

```diff
@@ -128,6 +130,10 @@ func resourceComputeInstanceGroupCreate(d *schema.ResourceData, meta interface{}
 		instanceGroup.NamedPorts = getNamedPorts(v.([]interface{}))
 	}
 
+	if v, ok := d.GetOk("network"); ok {
+		instanceGroup.Network = v.(string)
```

Comment:
In other terraform resources, we have the network parameter be the name of the network and then do a lookup. For consistency, I think we should do that here too. See https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/resource_compute_instance.go#L479 for an example.